build.gradle

```diff
@@ -291,8 +291,7 @@ def whitelistTemplateFiles = [
         '**/pushToZirc.sql',
         '**/pullFromZIRC.pl',
         '**/pushToZirc.pl',
-        '**/orphanChecks.pl',                                       //server_apps/DB_maintenance
-        '**/fx_permission_check.pl',
+        '**/fx_permission_check.pl',                                //server_apps/DB_maintenance
```

Comment:
Should fx_permission_check.pl be here rather than one line down under DB_maintenance?

Reply:
Hmm, maybe my comments in this part of the code aren't very clear. I had intended the path on the right to indicate that the file on that line and all files below it (until the next comment) are in the same directory.

For example, in the following screenshot, the files `loadNewSNPs.sql` and `loadNewSNPAttrs.sql` are both in the same directory.

<img width="880" alt="Screenshot 2024-08-29 at 4 18 23 PM" src="https://github.com/user-attachments/assets/f114d1af-683b-42d7-84c9-f0df656d7057">

Since I removed orphanChecks.pl from the list of files that need the template processing, I had to move that comment down to the next file in the same dir (fx_permission_check.pl).

 I think I'm being consistent with the comments in this PR, but perhaps I should rearrange these comments to be more clear?

Reply:
Ah, I see. Word wrap made me misunderstand.